packages/frontend/lib/index.ts

```diff
@@ -88,7 +89,6 @@ export default class Nango {
         }
 
         this.hostBaseUrl = config.host.replace(/\/+$/, ''); // Remove trailing slash.
-        this.status = AuthorizationStatus.IDLE;
```

Comment:
I am pretty sure `status` was here for a good reason. Why is it mostly useless now?

Reply:
I'm not sure why it was useful tbh, it's some kind of locking mechanism, I feel like there is no benefit in erroring vs just reopening a popup.
Maybe @khaliqgant knows best?

Reply:
I don't recall any specific implementation detail as this state tracking goes back to the Pizzly days: https://github.com/triggerdotdev/Pizzly/blob/master/packages/frontend/lib/index.ts#L80

If you're now tracking this in a different way then would go forth in removing. Otherwise I would be reluctant to remove it unless we clearly see the cases where it was used previously are no longer valid / needed.

Reply:
I really don't think it's useful now. If I trust the code it was to prevent further message from the websocket after it's done, but with this PR I'm actually closing the connection at the end so I don't see any problem removing it

Reply:
Cool. Works for me

Reply:
I have the feeling it was useful, something to do with the the window closing detection feature but I cannot remember any details unfortunately, I haven't touched this code for more than a year.
Your call though. If I were you I would keep it but I trust your judgment anyway.

Reply:
I see, it's a problem if we have feeling but we don't know, I can't even properly put it back because I don't know what to fix :D 
I'll see in prod 🤞🏻